### ISSUE TITLE
jar for 10.0.0 doesn't contain any classes

### ISSUE THREAD

**Issue by mr-git** [julienrf/play-json-derived-codecs#82]
updated project in IDEA, IDEA shows content like:
![image](https://user-images.githubusercontent.com/8796159/118111622-a366a900-b3ec-11eb-844e-e29f3da0c82a.png)

scalac complains:
```text
[error] ... not found: object julienrf
[error] import julienrf.json.derived
```

Could it be that publishing failed somehow?

Jar file-size:
* 9.0.0 - 85 137 bytes
* 10.0.0 - 315 bytes

**Comment by julienrf** [comment]
Indeed. It’s not clear to me what happened because I couldn’t reproduce the issue. I’ve manually published version 10.0.1 that should contain the jar files.

**Comment by mr-git** [comment]
thank you for fast response! much appreciated!
